opentreemap/exporter/tasks.py

```diff
@@ -62,7 +67,31 @@ def extra_select_and_values_for_model(
         field_name = perm.field_name
         prefixed_name = prefix + field_name
 
-        if field_name.startswith('udf:'):
+        if field_name in _UDFC_FIELDS:
+            try:
+                field_definition_id = (UserDefinedFieldDefinition
+                                       .objects.
+                                       get(model_type=model,
+                                           instance_id=instance.id,
+                                           iscollection=True,
+                                           name=field_name[4:]).id)
+            except UserDefinedFieldDefinition.DoesNotExist:
+                continue
```

Comment:
Consider getting the field IDs using the `udf_defs` function from `object_caches` for better performance